.github/workflows/dapr-test.yml

```diff
@@ -240,6 +240,11 @@ jobs:
         if: env.TEST_PREFIX != ''
         run: |
           make build
+
+          if [ "${{ env.TARGET_OS }}" = "linux" ] && [ "${{ env.TARGET_ARCH }}" != "arm" ]; then
```

Comment:
No ARM support for mariner?

Reply:
Mariner only supports amd64 and arm64. No support for 32-bit OS’s.